src/App.tsx

```diff
@@ -50,15 +36,15 @@ export const App: React.FC = () => {
           <div className="tile is-parent">
             <div className="tile is-child box is-success">
               <div className="block">
-                <UserSelector value={author} onChange={setAuthor} />
+                <UserSelector value={author} />
               </div>
 
               <div className="block" data-cy="MainContent">
                 {!author && <p data-cy="NoSelectedUser">No user selected</p>}
 
-                {author && !loaded && <Loader />}
+                {author && !loading && <Loader />}
 
-                {author && loaded && hasError && (
+                {author && loading && err && (
```

Comment:
The condition here checks for `loading && err`, which means it will show an error message while loading is still true. Ensure that this is the intended behavior. Typically, errors are shown when loading has completed, i.e., `!loading && err`.